pages/incidents/index.js

```diff
@@ -1,41 +1,42 @@
+import React, { useContext } from "react";
 import styled from "styled-components";
+import IncidentForm from "../../components/incidentForm";
+import { useIncidents } from "../../context/listContext";
+
+function IncidentPage() {
+  const incidents = useIncidents().incidents;
+  const setIncidents = useIncidents().setIncidents;
+  console.log(incidents);
+  function appendListEntry(time, location, offender, incident) {
+    setIncidents((listEntry) => [
+      ...listEntry,
+      {
+        time,
+        location,
+        offender,
+        incident,
+        id: Math.random().toString().substring(),
```

Comment:
```suggestion
        id: Math.random().toString(),
```
Drop the substring if you don't expect to shorten the string.
While Math.random works well for generating pseudo random numbers, consider using packages like nanoid. It's unconventional to see numbers as IDs and it's more common to see a cryptic set of lowercase & uppercase letters and numbers as IDs, which is what packages like nanoid are great for.